drivers/i2c/i2c_lpc11u6x.c

```diff
@@ -314,6 +314,11 @@ static int lpc11u6x_i2c_init(const struct device *dev)
 		return err;
 	}
 
+	if (!device_is_ready(cfg->clock_dev)) {
+		LOG_ERR("clock control device not ready");
```

Comment:
Logging does not seem to be enabled for this file. Can this be actually compiled successfully?

Reply:
You're right, thanks. Fixed.